rocket.chat/logo.svg

```diff
@@ -1,87 +1,78 @@
+

```

Comment:
This blank line is causing the browser to be unable to render the svg file: https://rawgit.com/engelgabriel/docs/67d206b72baa925087d9442c5bc46eae91c30f94/rocket.chat/logo.svg